interface-definitions/service_webproxy.xml.in

```diff
@@ -8,6 +8,39 @@
           <priority>500</priority>
         </properties>
         <children>
+          <node name="acl">
```

Comment:
This may be subjective, but to me, "set ... acl" feels like "acl" is a tag node with named children, like "rule" in firewall and routing policy objects.

I understand your motivation: you want to group both options under a common subtree, and I agree this is a good idea. I just wonder if "acl" is the best name for that subtree.

Since there are, AFAIR, no other options like these, maybe we want to just put them under the top level, e.g., "safe-ports" and "ssl-safe-ports".

Reply:
Ok, I'll change the logic, and move forward for command `set service webproxy safe-ports <port_number>` and `set service webproxy ssl-safe-ports <port_number>`